5_playing_with_image_arrays/drawing_on_images/README.md

```diff
@@ -0,0 +1,15 @@
+# DRAWING SHAPES ON IMAGES
+
+There are many reasons for drawing shapes on images such as for selecting the Region of Interest in computer vision applications, for making annotations, etc. Such shapes can be drawn by using the OpenCV library functions as well.
+
+The first step is to initialise a black image with all pixel values set to 0 and pre-set dimensions.
+
+After that, a line, circle and rectangle are drawn using the OpenCV functions *line*, *circle* and *rectangle*.
```

Comment:
Here, the aim was to implement line, circle, and rectangle from scratch. No use of OpenCV function except image read/write in the whole topic